azkustodata/query/from_kusto.go

```diff
@@ -0,0 +1,66 @@
+package query
+
+// value.go provides methods for converting a row to a *struct and for converting KustoValue into Go types
```

Comment:
update this (I did not look at other files, but they can also be out of date)

Reply:
It's only for this file, which was taken from the old system.